examples/hello_world/ios/Runner/GeneratedPluginRegistrant.h

```diff
@@ -2,6 +2,8 @@
 //  Generated file. Do not edit.
 //
 
+// clang-format off
```

Comment:
What is the need for this? Are generated files passed through clang-format automatically?

Reply:
I can revert this, it was a mistake.  This is just because I ran this, and when building this file gets updated. We should probably go through and update all of the runners sometime so this doesn't happen.